### PR TITLE
Update CMK all forms by setting npip default

### PR DESCRIPTION
**Pull request by satishraj777**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

Creating this PR to update our Databricks quick start template - CMK all forms with exposing the enableNoPublicIP parameter to our customers.

### CODE REVIEW

**Comment by satishraj777** [comment]
Hi, I am writing this PR on behalf of my team in Azure Databricks at Microsoft. I agree to the CLA. My MS ID is v-sgandamuni@microsoft.com

@microsoft-github-policy-service agree company="Microsoft"

File: quickstarts/microsoft.databricks/databricks-workspace-with-customer-managed-keys-all-forms/azuredeploy.json [hunk 1]
```diff
@@ -8,6 +8,13 @@
...
         },
+        "enableNoPublicIp": {
+            "type": "bool",
+            "defaultValue": false,
+            "metadata": {
+              "description": "Specifies whether to deploy Azure Databricks workspace with Secure Cluster Connectivity (No Public IP) enabled or not"
+            }
+          },
         "workspaceName": {
             "type": "String",
```
Comment (mvvsubbu) [inline quickstarts/microsoft.databricks/databricks-workspace-with-customer-managed-keys-all-forms/azuredeploy.json:19]: Can you also check if other templates have this parameter hardcoded and update them as well?

File: quickstarts/microsoft.databricks/databricks-workspace-with-customer-managed-keys-all-forms/azuredeploy.parameters.json [hunk 2]
```diff
@@ -5,6 +5,9 @@
     "workspaceName": {
       "value": "default"
     },
+    "enableNoPublicIp": {
+      "value": "null"
```
[the remark this answers is not in the document]
Comment (satishraj777) [reply quickstarts/microsoft.databricks/databricks-workspace-with-customer-managed-keys-all-forms/azuredeploy.parameters.json]: > why value is null? it should default or true right?

Yes. I put the value as 'false' for now. The 'default' or 'null' value were causing build errors.

File: quickstarts/microsoft.databricks/databricks-workspace-with-customer-managed-keys-all-forms/azuredeploy.json (hunk 1 above)
Comment (satishraj777) [reply quickstarts/microsoft.databricks/databricks-workspace-with-customer-managed-keys-all-forms/azuredeploy.json:19]: Sure Subbu. Once I make sure the build is good for this commit, I will go ahead and update all other templates with same parameter configuration.

**Review by mvvsubbu — APPROVED** [review]
Looks good to me

**Pull request merged**